### PR TITLE
Fixes #5138 - ISE support policy

### LINKED ISSUE #5138: Question: Is PowerShell ISE actually no longer in developement?
Hello, I see it being said on a lot of blogs, which tend to favor VSCode, example:
https://adamtheautomator.com/how-to-make-visual-studio-code-look-and-behave-like-the-powershell-ise/

But I'm struggling to find any official statement to back this.  Are you able to confirm whether or not the ISE is no longer being worked on? 

The reason I ask is that the debugging and integrated console experiences are still night and day better with ISE.  I do like VS Code, but the PowerShell extension pales in comparison to ISE for stability and performance, but most notably with debugging and the integrated console.  VS Code wins in other areas such as PSScriptAnalyzer, for example, but where there is overlap, PowerShell ISE seems to win.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 91aaa090-5d02-dfce-f23d-4d37925175bb
* Version Independent ID: 5220c32c-b163-7cef-860b-194ea52d646d
* Content: [Introducing the Windows PowerShell ISE - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/components/ise/introducing-the-windows-powershell-ise?view=powershell-6)
* Content Source: [reference/docs-conceptual/components/ise/Introducing-the-Windows-PowerShell-ISE.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/components/ise/Introducing-the-Windows-PowerShell-ISE.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes #5138 - ISE support policy
Fixes [AB#1650132](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1650132)

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [x] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/docs-conceptual/components/ise/Introducing-the-Windows-PowerShell-ISE.md [hunk 1]
```diff
@@ -29,8 +36,8 @@ Key features in Windows PowerShell ISE include:
   SHIFT+ENTER.
```
Comment (davidsmatlak) [inline reference/docs-conceptual/components/ise/Introducing-the-Windows-PowerShell-ISE.md]: Use the `<kbd>` tags for Shift+Enter, F1, and F5.

File: reference/index.yml [hunk 2]
```diff
@@ -74,27 +74,27 @@ productDirectory:
...
+        - url: https://docs.microsoft.com/powershell/scripting/dsc/getting-started/lnxgettingstarted
           text: Get started with DSC for Linux
         - url: https://dsccommunity.org/
           text: DSC Community
     # Card 5
     - title: Developer resources
       imageSrc: https://docs.microsoft.com/media/common/i_code-edit.svg
       links:
-        - url: https://docs.microsoft.com/powershell/developer
+        - url: https://docs.microsoft.com/powershell/scripting/developer
```
Comment (davidsmatlak) [inline reference/index.yml]: fyi: link doesn't work, but is part of the change, correct?

File: reference/index.yml [hunk 3]
```diff
@@ -106,7 +106,7 @@ productDirectory:
     - title: Related technologies
       imageSrc: https://docs.microsoft.com/media/common/i_tools.svg
       links:
-        - url: https://docs.microsoft.com/powershell/wmf/
+        - url: https://docs.microsoft.com/powershell/scripting/wmf/
```
Comment (davidsmatlak) [inline reference/index.yml]: fyi: link doesn't work, but is part of the change, correct?

File: reference/docs-conceptual/whats-new/What-s-New-in-the-PowerShell-50-ISE.md [hunk 4]
```diff
@@ -6,9 +6,18 @@ title:  What's New in the PowerShell 5.0 ISE
 
 # What's New in the Windows PowerShell 5.0 ISE
 
-This topic explains the new and updated features that have been introduced in versions of Windows
+This topic explains the new and updated features that have been introduced in version 5.0 of Windows
```
Comment (davidsmatlak) [inline reference/docs-conceptual/whats-new/What-s-New-in-the-PowerShell-50-ISE.md]: change to: version 5.0 of the Windows...

File: reference/index.yml (hunk 3 above)
Comment (sdwheeler) [reply reference/index.yml]: Fixed

File: reference/index.yml (hunk 2 above)
Comment (sdwheeler) [reply reference/index.yml]: fixed

File: reference/docs-conceptual/whats-new/What-s-New-in-the-PowerShell-50-ISE.md (hunk 4 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-the-PowerShell-50-ISE.md]: fixed

File: reference/docs-conceptual/components/ise/Introducing-the-Windows-PowerShell-ISE.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/components/ise/Introducing-the-Windows-PowerShell-ISE.md]: Fixed all key press instances

**Pull request merged**